extensions/parquet/base/src/main/java/io/deephaven/parquet/base/PlainIntChunkedWriter.java

```diff
@@ -9,30 +9,30 @@
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
-import java.nio.*;
+import java.nio.IntBuffer;
 
 
 /**
- * Plain encoding except for booleans
+ * Plain encoding except for ints
+ * TODO see if we can replicate from this, dhe seems to
  */
 public class PlainIntChunkedWriter extends AbstractBulkValuesWriter<IntBuffer, Number> {
     private final ByteBufferAllocator allocator;
     private final int originalLimit;
 
-    private IntBuffer targetBuffer;
-    private ByteBuffer innerBuffer;
+    private final IntBuffer targetBuffer;
+    private final ByteBuffer innerBuffer;
 
     PlainIntChunkedWriter(int pageSize, ByteBufferAllocator allocator) {
-        innerBuffer = allocator.allocate(pageSize);
+        this.innerBuffer = allocator.allocate(pageSize);
```

Comment:
Why are we making this code worse with unnecessary `this.` prefixes?
If you wanted, you could put the `this.allocator` assignment first, which would make the subsequent derived initializations look cleaner.

Reply:
Because the more similar the code is between dhc and dhe, the easier cherry-picking will be - that's the purpose of the commit.

Reply:
It's not worth it. Cherry-picking is beyond impossible for most of the code base already. We show our code to outsiders. Better to clean this up and suggest DHE make the same change.

Reply:
ok, i'll include this in my followup patch for dhe